src/main/java/jenkins/plugins/mattermost/StandardMattermostService.java

```diff
@@ -29,7 +29,7 @@
 	public StandardMattermostService(String endpoint, String roomId, String icon) {
 		super();
 		this.endpoint = endpoint;
-		this.roomIds = roomId.split("[,; ]+");
+		this.roomIds = roomId.split("[,;]+");
```

Comment:
Well, it's just hard to allow spaces as parts of bot names AND as channel name dividers. The above regex won't work in this case.

We have something like 'Bot Name 1@channel1,Bot Name 2@channel2'. Allowing spaces as dividers, the plugin wouldn't know whether to post to channel Bot or if it belongs to the name of a bot.

But afaik the description text for the channels already says 'divided by comma'. I'm not sure since when this description exists. (Didn't check the history yet.)